salt/modules/boto_route53.py

```diff
@@ -548,27 +548,27 @@ def get_record(
     """
     Get a record from a zone.
 
-    CLI Example:
+     CLI Example:
```

Comment:
same here, the entire doc for this function has been shifted. I believe it needs to align with the `"""` above.